source/admin_manual/list_of_events.rst

```diff
@@ -1791,3 +1795,79 @@ happens.
 +-----------------------+------------------------------------------------------+
 | object_id             | Object ID in the storage                             |
 +-----------------------+------------------------------------------------------+
+
+Dictionaries
+============
+
+.. versionadded:: 2.3.11
+
+Events emitted by dictionary drivers and dictionary server.
+
+.. _event_dict:
+
+Common fields for dict events
+-----------------------------
+
++----------------+------------------------------------------------------+
+| Field          | Description                                          |
++================+======================================================+
+| driver         | Name of the dictionary driver                        |
++----------------+------------------------------------------------------+
+| key            | Affected key or prefix                               |
++----------------+------------------------------------------------------+
+| key_found      | If the key was found or not                          |
```

Comment:
And more clearly that it's "yes" or "no". Although I'm still not sure if we should make it so explicit. I think generally the events so far have been "foo=yes" vs "foo isn't set at all". And the key_found=yes is the most normal situation.. Could this just be e.g. "notfound=yes" when it doesn't exist and nonexistent when it does?..

Reply:
eh, this is actually how it works. i just wrote it badly.

Reply:
Well, it's now key_found=yes vs nonexistent. But I was also thinking maybe the reverse would make more sense, since most of the time the key is found so it would reduce the event sizes a bit. "nonexistent=yes" sounds rather nice to me.

Reply:
i don't like `nonexistent`, but i'll add `key_not_found`, because `nonexistent` does not actually tell what is nonexistent. =)